string camera_topic
bool enabled
---
bool success
string message